player/tests.py

```diff
@@ -7,45 +6,29 @@
 @pytest.mark.django_db
 class TestPlayerModel:
 
-    @pytest.fixture
-    def user(self):
-        return get_user_model().objects.create_user(
-            username='testuser',
-            password='testpass'
-        )
+    def test_get_player(self, player):
+        player_from_db = Player.objects.get(pk=player.pk)
+        assert player_from_db == player
 
-    @pytest.fixture
-    def saved_player(self, user):
-        player = Player.objects.create(
-            user=user,
-            birth_date='1990-01-01',
-            favorite_ball_game=BallGame.Soccer
-        )
-        return player
-
-    def test_get_player(self, saved_player):
-        player = Player.objects.get(pk=saved_player.pk)
-        assert player == saved_player
```

Comment:
What's the reason for deleting this test case?

Reply:
what case you are reffering to ?

Reply:
`test_get_player()`

Reply:
Now I see it above. Never mind.